python/lsst/ts/standardscripts/base_track_target.py

```diff
@@ -0,0 +1,185 @@
+
+    def __init__(self, index, descr):
+        super().__init__(index=index, descr=descr)
+
+        self.config = None
+
+        # Flag to monitor if tracking started for cleanup task.
+        self.tracking_started = False
+
+        # Flag to specify which type of slew will be performend:
+        # slew_icrs or slew_object
+        self.slew_icrs = False
+
+    @property
+    @abc.abstractmethod
+    def tcs(self):
+        raise NotImplementedError()
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            $id: https://github.com/lsst-ts/ts_standardscripts/base_slew.yaml
+            title: BaseSlew v1
+            description: Configuration for BaseSlew
+            type: object
+            properties:
+              ra:
+                description: ICRS right ascension (hour)
+                anyOf:
+                  - type: number
+                    minimum: 0
+                    maximum: 24
+              dec:
+                description: ICRS declination (deg)
+                anyOf:
+                  - type: number
+                    minimum: -90
+                    maximum: 90
+              rot_value:
+                description: >-
+                  Rotator position value. Actual meaning depends on rot_strategy.
+                type: number
+                default: 0
+              rot_strategy:
+                description: Rotator strategy.
+                type: string
+                enum: ["sky", "parallactic", "physical_sky"]
+                default: sky
+              target_name:
+                description: Target name
+                type: string
+            if:
+              properties:
+                ra:
+                  const: null
+                dec:
+                  const: null
+              required: ["target_name"]
+            else:
+              required: ["ra", "dec"]
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    async def configure(self, config):
+        """Configure the script.
+
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Configuration
+        """
+
+        self.config = config
+
+        if hasattr(self.config, "ra"):
+            self.slew_icrs = True
+
+        if self.config.rot_strategy != "sky":
+            # TODO: Implement other rotation strategies (DM-26321).
+            raise NotImplementedError(
+                f"(DM-26321): {self.config.rot_strategy} not implemented. Use sky."
+            )
+
+    def set_metadata(self, metadata):
+        """Compute estimated duration.
+
+        Parameters
+        ----------
+        metadata : `Script_logevent_metadata`
+        """
+        metadata.duration = 1
+
+    async def run(self):
+
+        target_name = getattr(self.config, "target_name", "slew_icrs")
```

Comment:
Do you want to update the default target name, or are you still happy with "slew_icrs"?

Reply:
I think `slew_icrs` is a good default... After all, if the user does not provide a target name, it will operate in `slew_icrs` mode.

Reply:
Great. Thanks.